CODE_OF_CONDUCT.md

```diff
@@ -1,13 +1,13 @@
 # Code of Conduct
 
-We are a community based on openness and friendly, didactic, discussions.
+We are a community based on openness, friendly, and didactic discussions.
```

Comment:
I don't think this retains the same meaning

Reply:
Hi @jnothman thanks for the review. Could you please suggest the modifications.

Reply:
We are a community based on openness, as well as friendly and didactic discussions. 

Or the original sentence.